cmake/Platform/Common/runtime_dependencies_common.cmake.in

```diff
@@ -13,6 +13,7 @@ function(ly_copy source_file target_directory)
     if(NOT ${same_location})
         file(LOCK ${target_directory}/${target_filename}.lock GUARD FUNCTION TIMEOUT 300)
         if("${source_file}" IS_NEWER_THAN "${target_directory}/${target_filename}")
+            message(STATUS "Copying \"${source_file}\" to \"${target_directory}\"...")
```

Comment:
Is this debug code?

Reply:
it will be output during the build, only for files actually being copied
I added it here just to make it symmetric with mac, we are putting that status in mac and it has been helpful when the postbuild step fails. If it fails, the message shows what was tried to be copied when it failed.